app/src/main/java/org/mozilla/fenix/settings/address/AddressEditorFragment.kt

```diff
@@ -49,4 +50,9 @@ class AddressEditorFragment : SecureFragment(R.layout.fragment_address_editor) {
         super.onResume()
         showToolbar(getString(R.string.addresses_add_address))
     }
+
+    override fun onStop() {
```

Comment:
I see that we do this in `onPause` in the `CreditCardEditorFragment`. https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorFragment.kt#102

Do you have any thoughts on whether this should be done on pause or stop? I would prefer if we align both of them.

Reply:
A quick search for `view?.hideKeyboard()` also showed me that we typically call this in `onPause()` if that makes a difference.

Reply:
I know it is used in `onPause` in other places, but I attribute that to the fact that other "cleaning" operations are made at that point and it has been added to the bunch. I do not see any reason to have it in `onPause` having it fire multiple times when it is not necessarily needed (putting the fragment in the background and re-entering it would bring back the keyboard anyway). IMO the optimal place to call this is `onStop`, but I do not have anything against moving it to `onPause`.